spec/models/spree/payment_method/sisow_billing/sisow_payment_method_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Spree::PaymentMethod::SisowBilling::SisowPaymentMethod, type: :model do
+  let(:sisow_transaction) { double("Spree::SisowTransaction") }
+
+  it "should respond with false when calling payment_profiles_supported?" do
+    expect(subject.payment_profiles_supported?).to be false
+  end
+
+  it "should respond with true when calling auto_capture?" do
+    expect(subject.auto_capture?).to be true
+  end
+
+  it "should respond with true when the transaction was successfull" do
+    allow(sisow_transaction).to receive_message_chain(:status, :downcase).and_return('success')
+    expect(subject.purchase('123', sisow_transaction, {}).success?).to be true
+  end
+
+  it "should respond with false when the transaction was unsuccessfull" do
+    allow(sisow_transaction).to receive_message_chain(:status, :downcase).and_return('expired')
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.